DPG reference catalog for open standards

```diff
@@ -0,0 +1,54 @@
+###DPG Reference catalog for open standards
+##This reference catalog is an overview of standards that are recommended for the Digital Public Goods. The standards can be technical or semantic, and are sorted by relevant application. 
+
+#Datasets:
+JSON (JavaScript Object Notation)
+XML
+
+#Character encoding and languages:
+UTF-8, ISO/IEC 10646
+ISO 639-1/639-2/639-3
+
+#Web standards:
+HTML 
+CSS
+SSL
+HTTP
```

Comment:
Related to the SSL comment, browsers are starting to offer HTTPS-only modes and with the rise of readily-available certificates through services such as Lets Encrypt, the use of non-encrypted HTTP is becoming more and more discouraged. https://https.cio.gov/everything/

Reply:
FYI @nathanbaleeta